### DISCUSSION TITLE
Is DrawIO Desktop supported on Windows Server 2012R2?

### DISCUSSION THREAD

**Discussion by RichardMotaclan** [jgraph/drawio-desktop#1469] [Q&A]
We have installed the Windows Desktop version of DrawIO on a Windows Serrver 2012R2 remote desktop server and we get the following error when trying to run the program.
![image](https://github.com/jgraph/drawio-desktop/assets/146099134/689034fc-1082-43c6-9619-bde565082ee5)

**Comment by davidjgraph** [comment]
That is not a supported operating system.
